### ISSUE TITLE
Discussion: Preparing Miners for Filecoin Plus

### ISSUE THREAD

**Issue by zixuanzh** [filecoin-project/notary-governance#8]
## Goals
- Provide guidelines/suggestions for miners interested in storing Filecoin Plus client data.
- Start community discussion on how miners can differentiate and make themselves more attractive and discoverable.

## Background
- Filecoin is about reliable and useful storage and block reward aims to subsidize that.
- 10x increase in storage power for Filecoin Plus client deals is to subsidize tools, applications, features, and all the extra miles that will be developed on Filecoin, thus rewarding miner alignment with the network's mission and facilitating a sustainable economy.
- As the community collectively develops Filecoin Plus, what can a miner do to help the network and themselves succeed?

## Recommendations

- First of all, miners can and should leverage existing relationships to bring more high quality clients to the network, help them become Filecoin Plus clients, store and show case use cases if possible.
- Trust needs to be earned over time. To start, it will be easier if data is public (even though depending on the use cases, it may not always be the case but having something public makes it more easily justifiable for the clients, notaries, and the greater community. On top of that, it will be a window to showcase what Filecoin (and also the miner storing the data) is capable of and it will benefit everyone in the community.
- Miners should thus consider some ways of showcasing the storage quality, features, maybe have a sample demo page.
- Next, what are the differentiating features and services that a particular miner can offer, redundancy, encryption, fast retrieval, databases sitting on top of the storage provided etc. If there is any technology that a miner develops and is proud of, it will be great to release a less advanced but open source version. That will earn a miner greater reputation, attract more clients, and make Filecoin and hence the miner themselves more successful.
- Additionally, Miners should take steps to prepare for onboarding of client data - including considering creating a Terms of Service (template [here](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)) and a content policy (template [here](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md)). Note: Chinese translations are in progress and will be included in the Murmuration Labs filecoin-node-operator-kit.
- Last but not least, part of the [Filecoin Plus Principles](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0003.md) is to make the network more decentralized across geographies and use cases. Think about where miners are under represented at the moment, consider setting up an operation there. If you are already there and doing great, make sure the community knows about you.
- In short of a miner discovery platform at the moment, miners can consider advertizing themselves here. A possible template could be as follows and please feel free to make suggestions.
  - Miner ID
  - Location
  - (if applicable) Link to Terms of Service
  - (if applicable) Link to Content Policy
  - Contact Info (Slack, Website, Email)
  - Features
  - Special Offering

**Comment by rjan90** [comment]
Starting off this, creating a temporarily Miner template:

**MinerID**

**Comment by rjan90** [comment]
Starting off this issue:

**MinerID**
f023467

**Location**
Oslo, Norway

**Term of Servcie**
_Link Coming_

**Content Policy**
_Link Coming_

**Contact Information**
Slack: Phi-rjan
Website: [https://phimining.io/](https://phimining.io/)
Email: contact@phimining.io

**Features:**
Servers housed in a tier 3 datacenter in Oslo, Norway. Sub millisecond latency to the most densely populated areas of Norway.

**Special Offering**
0 FIL for verified data

**Comment by s0nik42** [comment]
**Miner ID**
f010479
**Location**
France Europe
**Link to Terms of Service**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)
**Link to Content Policy**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md)
**Contact Info** 
Slack : @s0nik42
Email : online@noelnoel.org
**Features**
- Fast retrieval
- 2TiB daily sealing power
- 2GB connectivity throughput 

**Special Offering**
Free for the first verified clients

**Comment by benjaminh83** [comment]
**Miner ID**
f02576
**Location**
Denmark Europe (Copenhagen area)
**Link to Terms of Service**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)
**Link to Content Policy**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)
**Contact Info**
Slack : **@BenjaminH**
**Features**
1 Gbit Fiber, Static IP behind NextGen Firewall, No VPN.
1.5TiB daily sealing power
Running on 75-95% renewable energy
Used energy is recirculated for heating
Contributor to Calibration/SR1/SR2 and multiple slack community champion
**Special Offering**
Free storage for the first verified clients

**Comment by mxz9102** [comment]
Miner ID
f09848

Location
Southern California USA

Link to Terms of Service
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)
Link to Content Policy
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)

Contact Info
Slack : @BigBear
Email : mzhang@sea-mounts.com

Features
1 Gbit Fiber, Static IP.
2 TiB daily sealing power

Special Offering
Free for the first verified clients.

**Comment by stuberman** [comment]
**Miner** ID
f01278

**Location**
Michigan, USA

**Contact Info**
Slack : @stuberman

**Features**
1 Gbit Fiber
Min size 100 KiB
Max size 32 GiB
160 GiB daily sealing power
0.0000000000001 FIL GiB/epoch

**Comment by Shekelme** [comment]
**Miner ID**
f08240

**Location**
Novosibirsk, Russia

**Contact Info**
Slack: @Rabinovitch
e-mail: aleksei.shcherbina@yandex.ru

**Features**
100 Mbit PON (can change to 1 Gbit if necessary)
UPS protected operation
Min size 256 B
Max size 32 GiB
0.0000000005 FIL GiB/epoch for Verified clients

There are also some miners in Russia and Ukraine I can control: f079817, f080480.

**Comment by flyworker** [comment]
**Miner ID**
f01276
f02401
f02387
f010194

**Location**
Montreal, Canada

**Contact Info**
website: https://www.nbfspool.com
Slack: @Charles Cao - NBFS
e-mail: contact@nbai.io

**Link to Terms of Service**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)
Link to Content Policy
 [Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)

**Features**

- 1 Gbit Fiber
- Tier3 Data Center
- Security staff on-site 24x7x365, Concurrently maintainable 1MW N+1 UPS Architecture. Flexible 2N solution available, minimum of 72 hours of on site fuel storage at full load with diverse fueling contractor arrangements
- 10TiB daily sealing power
- Contributor to Calibration/SR1/SR2/Slinghot and multiple slack community champion
- China node on demand

Price per GiB: 0.0000001 FIL
Verified Price per GiB: 0.0000001 FIL
Max Piece size: 8 GiB

Floating fees apply when base fee above 1 nano Fil

**Comment by ypool-xiaoyaozi** [comment]
f01800

**Comment by ypool-xiaoyaozi** [comment]
f068096

**Comment by cwhiggins** [comment]
**Miner ID**
f015927

**Location**
Long Island, NY, USA

**Contact Info**
Slack : @chris
chris@creativedigitalindustries.com

**Link to Terms of Service**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)


**Features**
1 Gbit Fiber
UPS protected system
Max size 32 GiB
750 GiB daily sealing power
0.000000000001 FIL GiB/epoch

**Comment by s0nik42** [comment]
@zixuanzh @jnthnvctr 

If it's not the right place lets move this somewhere else. I've got a few remarks/questions based on my reading of the 2 documents for mmlabs. 

# Terms of Service  :

My understanding was that a key reason to create the notary role  is to avoid the risk that miners deals with themselves to leverage the x10 factor by storing irrelevant data or garbage to themselves or friends.
Part of the Terms of Services, I think we should add 3 things  :
	- something saying client should not have a common interest with any miners they are storing to.
	- Something saying that in case of encrypted data, the key to decrypt the CID arbitrary selected by the notary should be provided to the notary by the client
	- Something saying that leveraging F+ for gaming filecoin network is forbidden, and actions (legal/technical) could be encountered against them.  

# content policy : 

Going through the content policy guidelines. A few things come to my mind :

- It appears that we need  granularity down to the “files” we host, not the “sector” as it is today. If one sector contains illegal and legal files; we will be stucked between disabling an illegal file and breaching a perfectly legal hosting contract (and probably pay the  fees, and loose reputation …) . 
- Auditing : we need a way to browse the sectors content offline (without having to download it and pay the fees/gas)
- A bit like notary, shall we have to think about having a role in charge of deleting/disabling a file across the whole network , whithout fees. 
- We host files that are accessible by all countries, with a huge variety of law. A file can be illegal in one country and legal in another one. Which law apply ? The client one ?  the miner one ?

**Comment by Acefighter-Au** [comment]
Miner ID
f010424

Location
Sydney, Australia

Link to Terms of Service
Standard FilecoinPlus
Link to Content Policy
Standard FilecoinPlus

Contact Info
Slack : @Steve T
Email : IPFS@flywheelmarketing.com.au

Features
200Mbps Tier 1 Internet Link (only 4 carrier in Australia)
2 dedicated links with automatic failover capability.
Fully redundant Carrier-grade IP Transit with diverse building entry  
The most highly peered network in Australia (1,300 BGP adjacencies) Access to global content with low latency
Data Centre – Certifications include: ISO 9001:2015, ISO/IEC 27001:2013 Certificate and PCI-DSS Compliant.
64 TiB daily sealing power

Special Offering

**Comment by Reiers** [comment]
**TechHedge Mining**

**MinerID**
[f022352](https://filfox.info/en/address/f022352)

**Location**
Borgen, Norway (Near Oslo)

**Terms of Service**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)

**Content Policy**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md)

**Contact Information**
Slack: @reiers
Website: _soon_
Email: nicklas@filecoin.no

**Features:**

- 2 TiB daily Sealing Power
- 1 Gbit Fiber, Static IP, no VPN. 
- 24/7 - Staff

**Special Offerings**
0 FIL for verified data

**Comment by ffteng47** [comment]
LuoliangTech Mining

MinerID
f04443

Location
ShangHai,China

Terms of Service
Standard FilecoinPlus

Content Policy
Standard FilecoinPlus

Contact Information
Slack: @jerrify
Website: www.luoliangtech.cn
Email: 253528688@qq.com

Features:
double 300Mbps Fiber
Max size 32 GiB
650 GiB daily sealing power
0.000000000001 FIL GiB/epoch

**Comment by TippyFlitsUK** [comment]
**Miner ID**
[f08403](https://filfox.info/en/address/f08403)

**Location**
Oxford, United Kingdom, Europe

**Link to Terms of Service**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)

**Link to Content Policy**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md)

**Contact Info**
Slack : @TippyFlits

**Features**

- 1 Gbit Fiber, No VPN
- Fast retrieval
- Min size 128 KiB
- Max size 32 GiB
- 420 GiB daily sealing power
- 0.00000001 FIL GiB/epoch for verified data

**Comment by embeded2017** [comment]
ABT Tech Mining

MinerID
f05664

Location
ShenZhen,China

Terms of Service
Standard FilecoinPlus

Content Policy
Standard FilecoinPlus

Contact Information
Slack: @tao
Email: 523613585@qq.com

Features

80M bps Tier 1 Internet Link, No VPN
Fast retrieval
Min size 1GiB
Max size 32 GiB
1T GiB daily sealing power
0.0000001 FIL GiB/epoch for verified data

**Comment by ypool-xiaoyaozi** [comment]
Miner ID
f068096
f01800

Location
Chengdu China

Link to Terms of Service
Standard FilecoinPlus

Link to Content Policy
Standard FilecoinPlus

Contact Info
Slack 1558249209@qq.com
Email: 1558249209@qq.com

Features

1 Gbit Fiber, No VPN
Fast retrieval
Min size 128 MiB
Max size 32 GiB
1T B daily sealing power
0.00000001 FIL GiB/epoch for verified data

**Comment by 18915855696** [comment]
ABT 技术挖掘

MinerID
f024156

地点
深圳，中国

服务条款
标准文件科林加

内容策略
标准文件科林加

联系信息松
弛： @ipfsnets
电子邮件： 155132990@qq.com

特征

80M bps 第 1 层
互联网链路
，无 VPN 快速
检索最小大小 1GiB 最大尺寸 32 GiB
1T GiB 每日密封功率
0.0000001 FIL GiB/epoch 验证数据

**Comment by 18915855696** [comment]
ABT Tech Mining

MinerID
f024156

Location
ShenZhen,China

Terms of Service
Standard FilecoinPlus

Content Policy
Standard FilecoinPlus

Contact Information
Slack: @ipfsnets
Email: 155132990@qq.com

Features

80M bps Tier 1 Internet Link, No VPN
Fast retrieval
Min size 1GiB
Max size 32 GiB
1T GiB daily sealing power
0.0000001 FIL GiB/epoch for verified data

**Comment by 18915855696** [comment]
zhirong Tech Mining

MinerID
f024156

Location
ShenZhen,China

Terms of Service
Standard FilecoinPlus

Content Policy
Standard FilecoinPlus

Contact Information
Slack: @ipfsnets
Email: 155132990@qq.com

Features

80M bps Tier 1 Internet Link, No VPN
Fast retrieval
Min size 1GiB
Max size 32 GiB
1T GiB daily sealing power
0.0000001 FIL GiB/epoch for verified data

**Comment by kimimhong** [comment]
MinerID
f020385

Location
Busan, Korea

Terms of Service
Standard FilecoinPlus

Content Policy
Standard FilecoinPlus

Contact Information
Slack: @김임홍
Email: daniel@codberg.com

Features

No VPN
200M bps internet / 40G network
Fast retrieval
Min size 512MiB
Max size 32 GiB
10TiB daily sealing power
0.0000001 FIL GiB/epoch for verified data

**Comment by Firefly2020-cloud** [comment]
Miner ID：f02420
Location：Hangzhou， China
 Terms of Service：Standard FilecoinPlus
 Content Policy:Standard FilecoinPlus
Contact Info
 Slack：@Nickole
Email：Tech@fireflyblock.com
Features:
100M bps Tier 1 Internet Link, No VPN
Fast retrieval
Min size :256MB
Max size :32GiB
1T GiB daily sealing power
0.0000001 FIL GiB/epoch for verified data

**Comment by namoo1** [comment]
MinerID
f03488, f024184, f033356

Location
Korea

Company
we4u

Terms of Service
Standard FilecoinPlus

Content Policy
Standard FilecoinPlus

Contact Information
Slack: MeestNam
Email: weforuplus

Features

- No VPN, Static IP
- 1G bps internet
- Fast retrieval
- Ask: f03488, f024184, f033356
- Price per GiB: 0.0000001 FIL
- Verified Price per GiB: 0.000000000000000001 FIL
- Max Piece size: 32 GiB
- each of  4TiB daily sealing power

Reputation f03488 
**90**
[](https://storage.codefi.network/)

**Comment by funktafide** [comment]
Miner ID
f014768

Location
Singapore

Contact Info
Slack : @Funk
Email: afunkhouser@fastmail.fm

Features
1 Gbit Fiber
Min size 128 KiB
Max size 32 GiB
160 GiB daily sealing power, depending in mpool
0.000000001 FIL GiB/epoch

**Comment by jamerduhgamer** [comment]
**Miner ID**
f066596

**Location**
California, USA

**Link to Terms of Service**
Standard FilecoinPlus

**Link to Content Policy**
Standard FilecoinPlus

**Contact Info**
Slack: @kemihu
Email: piknik@piknikinc.com

**Features**
- 10 TiB daily sealing power and increasing
- 1 Gbit Fiber, Static IP, No VPN
- 24/7 on site staff
- UPS Protected Systems

**Special Offering**
Free storage for the first 1000 verified clients

**Comment by neondragon** [comment]
**Miner ID**
f019551

**Location**
Birmingham, United Kingdom

**Terms of Service**: [Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)
**Content Policy**: [Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md). Content must comply with UK law.

**Contact Info**
Slack: [@NeonixAF f019551](slack://user?team=TEHTVS1L6&id=U01ACMASDEW)

**Features**
- 1Gb/s bandwidth
- Data stored with multiple copies, including regular scans to detect and correct errors
- Available over IPv4 and IPv6
- Physically located in the UK (No VPN). Ideally located for retrieval deals covering Western Europe and Eastern US.
- Backup electrical system

**Pricing**
Initially 0 FIL/GiB/Epoch for verified data.
After a trial period, pricing will be regularly adjusted to be fair and competitive in-line with Filecoin network fees.

**Comment by dgreuel2002** [comment]
**Miner ID**
f047419

**Location**
Wisconsin, USA

**Link to Terms of Service**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)

**Link to Content Policy**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md)

**Contact Info**
Slack : @DeeGee

**Features**

- 1 Gbit Internet, Ubiquiti Router
- Dell PowerEdge Fileservers with redundant raid 5 configuration in arrays of 6 disks
- Expandable storage rack
- Min size 4 GiB
- Max size 32 GiB
- 192 GiB daily sealing power
- 0.00000000001 FIL GiB/epoch for verified data for a trial period and then rates will be adjusted.

**Comment by jjxx2021** [comment]
**Miner ID**
f09569

**Location**
Beijing, China

**Terms of Service**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)

**Content Policy**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md)

**Contact Information**
Slack: @jjxx
Email: jjxx2021@163.com

**Features**

- 1Gbps Fiber, Static IP, no VPN
- 1TB daily Sealing Power
- UPS protected system
- Min sector size: 512MB
- Max sector size: 32GB

**Comment by dodohack** [comment]
**Miner ID**
[f081323](https://filfox.info/en/address/f081323)

Location
Milton Keynes, United Kingdom

Link to Terms of Service
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)

Link to Content Policy
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)

Contact Info
Slack : @Aries

**Features**

- 1 Gbit leased line Fiber, Static IP, no VPN, ~8ms latency to all UK network.
- UPS protected system
- Supermicro storage server with RAID 5 redundancy.
- Expandable storage rack
- Min size 4 GiB
- Max size 32 GiB
- 1.5 TiB daily sealing power
- 0.00000000001 FIL GiB/epoch for verified data for a trial period and then rates will be adjusted.

**Comment by acfreeman001** [comment]
##### Miner ID
f014569

##### Location
Yangzhou, China

##### Link to Terms of Service
Standard FilecoinPlus

##### Link to Content Policy
Standard FilecoinPlus

##### Contact Info
Slack : acfreeman

##### Features

* 1 Gbit Fiber
* Tier3+ Data Center
* More than 15 years of enterprise storage system development & maintenance experience
* Maintain more than 70 PiB storege power in Filecoin mainnet
* Top technical team to handle with techinical issues, including Filecoin, hardware, hard drives, OS and other issues
* 7*24 service
* Long term service: our goal is the marketing leader of decentralized storege provider, NOT just miners who want to make money quickly.

##### Price
Price per GiB: 0.000000001 FIL
Verified Data Price per GiB: 0 FIL
Price may be changed when the basefee is high. Let's discuss it if necessary.

**Comment by xinaxu** [comment]
**Miner ID**
f064218

**Location**
Washington, USA

**Link to Terms of Service**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)

**Link to Content Policy**
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md)

**Contact Info**
Slack : XinAn Xu

**Features**
1 Gbit Fiber
2TiB Daily sealing power
UPS protected system
24/7 monitoring and alerting
Storage redundancy to protect your data

**Price**
Price per GiB: 0.00000001 FIL
Verified Data Price per GiB: 0 FIL
Future price may be changed according to market condition and gas fees
Min size: 1 GiB
Max size: 32 GiB

**Comment by cryptowhizzard** [comment]
MinerID
f01240

Location
Amsterdam, the Netherlands

Company
[Speedium ](https://www.speedium.network)

Terms of Service
Standard FilecoinPlus

Content Policy
Standard FilecoinPlus

Contact Information
Slack: Wijnand Schouten
Email: contact@speedium.network

Features

No VPN, Static IP
10Gbps internet
Fast retrieval
Ask: f01240
Price per GiB: 0.0000001 FIL
Verified Price per GiB: 0.0000000005 FIL
Max Piece size: 32 GiB
4TiB daily sealing power

**Comment by leitaste** [comment]
**Miner ID**
f09652
f02614

**Location**
Sichuan China

**Contact Info**
website: https://www.atpool.com
Slack: @lei
e-mail: wanglei@atpool.com

**Link to Terms of Service**
Standard FilecoinPlus
**Link to Content Policy**
Standard FilecoinPlus

**Features**
1 Gbit Fiber
Tier3 Data Center
Construction of professional T3+ mine
Operation and maintenance staff on-site 7days and 24hours
UPS protected operation，and equipped with diesel generator
The operation and maintenance team has large-scale server maintenance and system level development experience

Price per GiB: 0.00000001 FIL
Verified Price per GiB: 0 FIL
Min size：128 MiB
Max size：32 GiB
*Future price may be changed according to market condition and gas fees

**Comment by quantumgw** [comment]
Miner ID：f018501

Location：Beijing China

Terms of Service：Standard FilecoinPlus

Content Policy:Standard FilecoinPlus

Contact Info
Slack：@yusun
Email：xueshengsy@gmail.com

Features:
100M bps static ip. No vpn
Ups and gasoline generator protection

Fast retrieval
Price per GiB: 0.0000001 FIL
Verified Data Price per GiB: 0.000000000001 FIL
Future price may be changed according to market condition and gas fees
Min size: 1 GiB
Max size: 32 GiB

**Comment by NSC-FIL** [comment]
Miner ID
f022142

Location
New York City, USA

Link to Terms of Service
Standard FilecoinPlus

Link to Content Policy
Standard FilecoinPlus

Contact Info
Slack : @Nelson
Comment: Actively monitoring Slack ... quick respond to Direct Messages and troubleshooting with clients.

Features
    200Mbps Internet, No VPN, Static IP
    Min size 1 GiB
    Max size 32 GiB
    ** Trial period: 0 FIL GiB/epoch for verified data!  Will increase by 2021. **

**Comment by lucas556** [comment]
Miner ID：f061158

Location：Zhengzhou, China

Terms of Service：Standard FilecoinPlus

Content Policy：Standard FilecoinPlus

Contact Information：
Email: 56923561@qq.com

Features:

- 100Mbps GPON Internet, No VPN, Static IP
- 2TiB Daily sealing power
- UPS protected system
- 24/7 monitoring and alerting
- Storage redundancy to protect your data
- Min size 256 B
- Max size 32 GiB
- Price per GiB: 0.0000000026 FIL
- Verified Price per GiB: 0 FIL

**Comment by Shekelme** [comment]
New growing Filecoin cluster is online!

Miner ID
f0187709

Link to Terms of Service
Standard FilecoinPlus

Link to Content Policy
Standard FilecoinPlus

Location
Moscow, Russia

Contact Info
Slack: @Rabinovitch
e-mail: aleksei.shcherbina@yandex.ru

Features
50 Mbit Internet channel (will be expanded in the near future, possibly taking into account the wishes of customers)
UPS protected operation
Min size 1GiB (where else can you find such a low value?)
Max size 32 GiB
0.00000002 FIL GiB/epoch
0.0000000044 FIL GiB/epoch for Verified clients
Raidz2 ZFS array to store your data.

The price may change depending on the current value of the basefee.

**Comment by Byeager1** [comment]
Miner ID:     f023971

Location:     United States

Link to Terms of Service:     https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md

Link to Content Policy:        https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md

Contact Info:     Slack:  @Brian Y - very active on slack and will respond quickly to DMs

Features:
500Mbit  Fiber Internet Service, Static IP
4 TiB of Daily Sealing Power Capacity 
Storage Capacity:     1.4 PiB
Min Size:     512 MiB
Max Size:     32GiB
Regular Deal Price:     0.00000001 FIL GiB/epoch
Verified Deal Price:     0 FIL 
Raidz3 ZFS array to store data 
Entire storage cluster housed in Tier3 Data Center
*Price may change in future subject to value of basefee

**Comment by livspect** [comment]
Miner ID
f0104671

Location
Tokyo, Japan

Link to Terms of Service
[Standard FilecoinPlus](https://github.com/livspect/filecoin-node-operator-kit/blob/main/terms-of-service.md)

Link to Content Policy
[Standard FilecoinPlus](https://github.com/livspect/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md)

Contact Info
Slack : @livspect
Email : info@livspect.com

Features
10 Gbps Fiber, Static IP, No VPN
300GiB or more daily sealing power

RAID10, SAS HDD
Planning to build the white data center that uses 100% green energy.

Min Size : 128 MiB
Max Size : 32 GiB

Special Offering
Free for verified clients.

**Comment by kimimhong** [comment]
MinerID
f020385

Location
Busan, Korea

Terms of Service
Standard FilecoinPlus

Content Policy
Standard FilecoinPlus

Contact Information
Slack: @김임홍
Email: daniel@codberg.com

Features

No VPN
200M bps internet / 40G network
Fast retrieval
Min size 512MiB
Max size 32 GiB
10TiB daily sealing power

**Comment by cloudC20** [comment]
MinerID
f033025
Location
Daegu, Korea

Terms of Service
Standard FilecoinPlus

Content Policy
Standard FilecoinPlus

Contact Information
Slack: @cloudC20

Features
1Gbit Internet service / Fast Ethernet
Min size 1MiB
Max size 32GiB
Verified 0 FIL
0.000000005FIL GiB/Epoch

**Comment by risc2000** [comment]
Miner ID
f0838467

Location
Gyeonggi-do, Korea

Link to Terms of Service
[Standard FilecoinPlus
](https://github.com/livspect/filecoin-node-operator-kit/blob/main/terms-of-service.md)
Link to Content Policy
[Standard FilecoinPlus
](https://github.com/livspect/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md)
Contact Info
Slack : @YM Kong
Email : allen@cobaltray.com

Features
40 Gbps/Redundant/UPS/FullRAID
14Tib or more daily sealing power
Controlled Temperature 

Min Size : 128 MiB
Max Size : 64 GiB

Special Offering
Free for verified clients.

**Comment by vpupkincanada** [comment]
Miner ID
[ f0167254](https://filfox.info/en/address/f0167254)

Location
Saint Petersburg, Russia

Link to Terms of Service
Standard FilecoinPlus

Link to Content Policy
Standard FilecoinPlus

Contact Info
Slack : @U026YGVESBG
e-mail: vpupkin.canada@gmail.com

Features

100 Mbit Internet channel.
UPS protected system
Supermicro storage server (Raidz2 ZFS, 3PiB) .
Min size 100 MiB
Max size 32 GiB
2 TiB daily sealing power

**Comment by ahn15e1a** [comment]
Miner ID
f0862933
Location
Charlotte, NC USA
Link to Terms of Service
Standard FilecoinPlus
Link to Content Policy
Standard FilecoinPlus
Contact Info
Slack : @Andy Nguyen
Features
* 600mb
* Fast retrieval
* Min size 32 MiB
* Max size 32 GiB
* 1tb GiB daily sealing power
* 0.000000001 FIL GiB/epoch for verified data

**Comment by zeyupan888** [comment]
MinerID
f022289

Location
China

Terms of Service
Standard FilecoinPlus

Content Policy
Standard FilecoinPlus

Contact Information
Slack: @Cabrina Huang
Website: soon
Email: cabrinahuang@gmail.com

Features:

5 TiB daily Sealing Power
1 Gbit Fiber, Static IP, no VPN.
24/7 - Staff
Tier-3 IDC
Special Offerings
0 FIL for verified data

**Comment by hFIL** [comment]
**Miner ID**
f0113008

**Location**
Hong Kong

**Contact Info**
GitHub: [https://github.com/hFIL/f0113008/issues](https://github.com/hFIL/f0113008/issues)

**Link to Terms of Service**: [Link](https://github.com/hFIL/f0113008/blob/main/tos.md)
**Link to Content Policy**: [Link](https://github.com/hFIL/f0113008/blob/main/Content-Policy-Guide.md)

Features
Small and Independent Miner
1Gbit Fiber
Minimal Downtime
Sectors backup with LTO Tape Drives

Price per GiB: 0.0000000004FIL/GiB/Epoch
***Verified Price per GiB: 0 FIL (Free!)***
Min size：32 MiB
Max size：32 GiB
*Future price may be changed according to market, network and chain conditions

**Comment by ly82505129** [comment]
**Miner ID**：f01509930
**Location**：China，Asia

**Link to Terms of Service**：Standard FilecoinPlus
**Link to Content Policy**：Standard FilecoinPlus

**Contact Info**
Slack : @allyn
Email : allyn0306@gmail.com

**Features**
Active member of venus community
Developing in Filecoin Ecosystems
Fast retrieval
2TiB daily sealing power
1GB connectivity throughput

**Comment by namoo-cloud** [comment]
Miner ID
f03488 / f024184 / f033356 

Location
Korea / Uwang

Link to Terms of Service
Standard FilecoinPlus

Link to Content Policy
Standard FilecoinPlus

Contact Info
Slack : @Meest Nam
Email : weforuplus@gmail.com

Features
1Gbps Fiber, Static IP, No VPN
30TiB or more daily sealing power

RAID6, SAS HDD

Min Size : 1 GiB
Max Size : 32 GiB

Special Offering
Free for verified clients.

**Comment by paar13kr** [comment]
Miner ID: f01764587, f01466075, f01790264
Location: Korea / Gyeonggi-do
Link to Terms of Service：Standard FilecoinPlus
Link to Content Policy：Standard FilecoinPlus

Contact Info
Slack: @Revolori
Email: scvpark@naver.com

Features
1Gbit Fiber / 40G network
Fast retrieval
Min size 100Mib
Max size 32 Gib
Price per GiB: 0.0000000004FIL/GiB/Epoch
Verified Price per GiB: 0 FIL
3Tib daily sealing

**Comment by Shekelme** [comment]
64GiB miner is waiting for your data!

Miner ID
[f01222595](https://filfox.info/en/address/f01222595)

Link to Terms of Service
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/sample-terms-of-service-template.md)

Link to Content Policy
[Standard FilecoinPlus](https://github.com/Murmuration-Labs/filecoin-node-operator-kit/blob/main/Content-Policy-Guide.md)

Location
Moscow, Russia

Contact Info
Slack: @Rabinovitch
e-mail: aleksei.shcherbina@yandex.ru

Features
200 Mbit Internet channel (will be expanded in the near future, possibly taking into account the wishes of customers)
UPS protected operation
Min size 1MiB (where else can you find such a low value?)
Max size 64 GiB
0.000000004 FIL GiB/epoch
0 GiB/epoch for Verified clients
Raidz2 ZFS array to store your data.

The price may change depending on the current value of the basefee.

**Comment by AleskeySidorkin** [comment]
Miner ID
[f0167254](https://filfox.info/en/address/f0167254)

Location
Saint Petersburg, Russia

Link to Terms of Service
Standard FilecoinPlus

Link to Content Policy
Standard FilecoinPlus

Contact Info
Slack : @U026YGVESBG
e-mail: sidorkin.alex007@gmail.com

Features

200 Mbit Internet channel.
UPS protected system
Supermicro storage server (Raidz2 ZFS, 3PiB) .
Min size 100 MiB
Max size 32 GiB
6 TiB daily sealing power

**Comment by dafsic** [comment]
Miner ID
f017665

Location
Shenyang, Liaoning, China

Link to Terms of Service
Standard FilecoinPlus

Link to Content Policy
Standard FilecoinPlus

Contact Info
Slack : @p0d347h
e-mail: dafsic@gmail.com

Features
1Gbit Fiber / 40G network
UPS protected system
Supermicro storage server (Raidz2 ZFS, 3PiB) .
Min size 100 MiB
Max size 32 GiB
10 TiB daily sealing power
Price per GiB: 0.000000005FIL/GiB/Epoch
Verified Price per GiB: 0 FIL

**Comment by dkkapur** [comment]
Closing this issue out for now. Those wanting to advertise to clients should get on https://github.com/filecoin-project/filecoin-plus-client-onboarding/blob/main/miners.md, or coordinate using client applications, #fil-deals-market, #fil-plus, etc.

**Comment by BATMAH69** [comment]
https://github.com/filecoin-project/filecoin-plus-client-onboarding/blob/main/miners.md ->
https://github.com/filecoin-project/filecoin-plus-client-onboarding/blob/main/storageproviders.md
